internal/consensus/reactor.go

```diff
@@ -453,6 +458,20 @@ func (conR *Reactor) subscribeToBroadcastEvents() {
 		}); err != nil {
 		conR.Logger.Error("Error adding listener for events (ProposalBlockPart)", "err", err)
 	}
+
+	err := conR.conS.evsw.AddListenerForEvent(
```

Comment:
Do you plan to add the FireEvent call for this event in the subsequent PR? Or does it make sense to do it in this PR? 
See https://github.com/cometbft/cometbft/blob/789cf37ba3692a9b8a666c753f66b281dcd46517/internal/consensus/state.go#L2461 and https://github.com/cometbft/cometbft/blob/789cf37ba3692a9b8a666c753f66b281dcd46517/internal/consensus/state.go#L1800 ( I imagine you need only the second case for Blobs).

Reply:
Ok sounds good, can you please in this PR just add a TODO here that there is no corresponding FireEvent. If someone else picks it up, they might overlook this